docs/book/src/gridlock.md

```diff
@@ -64,7 +64,14 @@ Divide into implemented or not.
 - hacks to allow conflicting turns at really broken intersections
 - manually timing signals
 - Last resort: if someone's waiting on a turn >5m, just go.
+
+## Not implemented
+
 - Dynamic rerouting
+- Allow multiple vehicles through intersection at once if there is enough space
```

Comment:
I was referring to behavior described in https://github.com/dabreegster/abstreet/issues/185 

Is it now supposed to be fixed?

Reply:
Hmm, at the time, I interpreted that bug to be about groups of lights close together. Could you point to a certain intersection ID and time? Wouldn't hurt to see exactly the case you're observing

Reply:
Intersection 13, weekday traffic Montlake and Eastlake has something similar.

![Peek 2020-07-20 17-24](https://user-images.githubusercontent.com/899988/87955554-332e1600-caae-11ea-9576-7bca147d45ad.gif)

I noticed that worst cases have trouble in form of "crossing + tiny road + crossing + plenty of space" where noticing that vehicle will certainly fit is trickier.

University district, crossing 110

![Peek 2020-07-20 17-29](https://user-images.githubusercontent.com/899988/87955894-a0da4200-caae-11ea-86c6-ef41530f327b.gif)

I do not remember tiny road in the original case but maybe I simply missed it.

Reply:
First issue is from the lane-changing model. I updated the [doc](https://github.com/dabreegster/abstreet/blob/master/docs/book/src/gridlock.md). The proper fix in that particular case is to not allow the second lane to turn right at all! It's hard to enforce that, but I have some possibly easy improvements to how vehicles adjust the lane they want to use last minute. I'll play around with those soon.

The second issue is because Lane #235 is tiny, so only one vehicle fits at a time. If you disabled block-the-box protection, this particular problem would go away. This will be handled once I figure out how to stretch uber-turns across intersections controlled by stop signs and make more parts of the sim layer understand this concept of multi-turn movements.